crates/rerun/Cargo.toml

```diff
@@ -31,19 +31,23 @@ glam = ["re_sdk?/glam"]
 ## Integration with the [`image`](https://crates.io/crates/image/) crate.
 image = ["re_log_types/image"]
 
-## Embed the Rerun SDK and re-export all of its public symbols.
-sdk = ["dep:re_sdk", "re_sdk?/re_viewer"]
+## Support a native viewer
+native_viewer = ["re_sdk?/native_viewer"]
 
 ## Support for running a HTTP server that listens to incoming log messages from a Rerun SDK.
 server = ["re_sdk_comms/server"]
 
+## Embed the Rerun SDK and re-export all of its public symbols.
+sdk = ["dep:re_sdk"]
```

Comment:
[not actionable in _this_ PR] calling this sdk here gives us a bit of friction: In the Docs we talk about the "Rerun SDK" as the union of API and viewer. It follows we should call it re_api 🤔